lm_eval/tasks/unitxt/20_newsgroups.yaml

```diff
@@ -1,3 +1,3 @@
-include: unitxt_tasks.classification.multi_class
 task: 20_newsgroups
```

Comment:
would naming these tasks with the prefix `unitxt_` be preferable for you for discoverability?

Reply:
I think it would be nicer for unitxt users to be able to add tasks which can be used by others without association to unitxt